python/python/glide/config.py

```diff
@@ -51,6 +51,14 @@ class ProtocolVersion(Enum):
     """
 
 
+class TlsMode(Enum):
```

Comment:
I think the name is confusing, it sounds like TlsMode should hold 'Disabled'/'Enabled' options, so the Auto option sounds like it choose automatically if to disable or to enable TLS. Maybe we can add 'Disabled' entry that will be set by default.

Reply:
How would this enum will be used if we would want to allow the user to pass certificates?

Reply:
we'd need to have an object to handle that. The core of this change is to change the value to not be a boolean. As long as we plan on supporting auto & enabled, in TS & Python we could add more object values on top.

Reply:
added "disabled" option to enum.